lib/migration-stream.js

```diff
@@ -72,6 +79,17 @@ function MigrationStream() {
       migrationStream.push(tile);
       return callback();
     }
+    else{
+      if(!v1TileDataLogged && process.env.LOG_V1_TILES){
+        v1TileData['tile_version'] = 'V1';
+        if (v1TileData.tile_version) {
+            console.log('[Debug] - begin writing v1-stats file');
+            fs.writeFileSync('v1-stats.json', JSON.stringify(v1TileData));
+            v1TileDataLogged = true;
+            return callback();
```

Comment:
I think we _don't_ want to return the callback here! If we have a v1 tile, we want to log it AND migrate it to a v2 tile. Otherwise this will result in a single v1 tile getting through without being converted. The callback will be executed here after the `migrate` method: https://github.com/mapbox/mapbox-tile-copy/pull/104/files#diff-4ff08a20a882a7df911bd7bdb3b11aa4R103

Perhaps this is why the tests are regularly failing, one tile isn't being pushed into the migrationStream?